osu.Game/Rulesets/Mods/ModSnapColour.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// Mod that colours hitobjects based on the musical division they are on
+    /// </summary>
+    public class ModSnapColour : Mod
+    {
+        public override string Name => "Snap Colour";
+        public override string Acronym => "SC";
+        public override LocalisableString Description => new LocalisableString("Colours hit objects based on the rhythm.");
```

Comment:
If you want to make this string localisable this is not the way to do it, please look at how it's done in the `osu.Game.Localisation` namespace/folder. Otherwise, you can just use a primitive string, there is an implicit type conversion, you'll be fine.

Reply:
Thank you, I just changed it to a primitive string for now, can always make it (properly) localisable later if needed

Reply:
Localisation for mod descriptions should probably be done on https://github.com/ppy/osu-web rather than here